client/src/components/about/index.jsx

```diff
@@ -190,7 +190,7 @@ const About = () => {
           </InfoDisplay>
         </Col>
       </Row>
-    </Container>
+    </>
```

Comment:
I imagine, we don't need a fragment here since there is an outer container. Doesn't need to be updated in this PR though.